samples/MeteoControl_mqtt/app/application.cpp

```diff
@@ -26,11 +24,9 @@ void init()
 
 	WifiStation.config(WIFI_SSID, WIFI_PWD);
 	WifiStation.enable(true);
-	//WifiStation.setIP(ESP_IP);
 	WifiAccessPoint.enable(false);
 	WDT.enable(false);	//disable watchdog
-	WifiEvents.onStationDisconnect(connectFail);
-	WifiEvents.onStationGotIP(gotIP);
+	System.onReady(systemReady);
```

Comment:
@zgoda Please, change System.onReady back to WifiEvents.onStationGotIP. In this example we only have Station connecting to AP.